src/app/features/vendor/models/vendor.model.ts

```diff
@@ -0,0 +1,12 @@
+export interface Vendor {
```

Comment:
Interface name should start with 'I'(IVendor here)

Reply:
updated